swagger/api.yaml

```diff
@@ -72,6 +72,45 @@ paths:
   "/enterprise/v3/enterprise-customer/{uuid}/learner-summary":
```

Comment:
Yeah looking at this swagger file, I can tell you that it is a completely separate specification from what we generate for studio. There are different needs here, as well. It's specifying the API for the API gateway, and then everything gets passed on to the studio backend APIs. These are two separate things and it would be a mistake to use the same swagger specification that we will have for studio also here.